controller/jokes.go

```diff
@@ -0,0 +1,40 @@
+package controller
+
+import (
+	"academy/services/dataload"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/gorilla/mux"
+)
+
+//JokesGetJokes all Jokes in the data
+func JokesGetJokes(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-type", "application/json")
+	w.WriteHeader(http.StatusOK)
+	json.NewEncoder(w).Encode(dataload.ReadData())
+}
+
+//JokesGetOneJoke only one joke by ID
+func JokesGetOneJoke(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	jokeID, err := strconv.Atoi(vars["id"])
+	if err != nil {
+		fmt.Fprintf(w, "Invalid ID")
```

Comment:
If some validation failed you would like to return and also set a header with a status code of 400(Bad request) or something appropriate

Reply:
I added the header and returned a JSON